todo-app/src/hooks/useFetch.tsx

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import IToDo from "../interfaces/Todo";
+
+const useFetch = (url: string) => {
```

Comment:
This hook will only work for get apis what about post,  put and delete ?

Reply:
working on it in the third module